library/SpamFilter/PanelSupport/Cpanel.php

```diff
@@ -2407,12 +2407,13 @@ public function migrateDomainsTo($params)
             $domains[] = $collectionDomain['name'];
         }
 
-        if (count($domains) > 0) {
+        $countDomains = count($domains);
```

Comment:
I noticed you checked is_array(x) or is_countable(x) in all other places, except here. I don't know if it is an issue, but just letting you know.

Reply:
That's because in this case $domains is an array built locally right before that count: 
```
        // Retrieve all domains
        $collectionDomains = $this->getCollectionDomains();
        $domains           = array();
        foreach ($collectionDomains as $collectionDomain) {
            $domains[] = $collectionDomain['name'];
        }

        $countDomains = count($domains);

```

Reply:
Ok, thanks for the clarification.